package.json

```diff
@@ -36,13 +36,14 @@
     "codegen"
   ],
   "dependencies": {
+    "@vectara/stream-query-client": "^3.0.0",
```

Comment:
I tried running everything locally without running `npm i` and things seemed to work fine. I think we can remove this?